test/mocha.opts

```diff
@@ -1,3 +1,3 @@
 --reporter spec
```

Comment:
While we are here, can we move from mocha.opts to a config file ? Looks like it is deprecated and will be removed in future: https://mochajs.org/#mochaopts. `.mocharc.js` looks fine by me.